bin/cli.js

```diff
@@ -30,5 +30,14 @@ var converterExtMap = {
 ;(function convert() {
     var code = fs.readFileSync(program.sourcePath, 'utf-8')
     var ext = path.extname(program.sourcePath)
-    console.log(converterExtMap[ext](code, program))
+    var jss = converterExtMap[ext](code)
+    /*
+    * program:
+    *   - `pretty` pretty formatting
+    *   - `export` namespace string to add before output
+    */
+    var spaces    = program.pretty ? '  ' : ''
```

Comment:
I use normal indentation, always the same amount of spaces. Will be easier to check once we have eslint in this repository too.